modules/aspect_rules_js/0.13.0/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
```

Comment:
Maybe also don't skip the regular test for building targets under `@aspect_rules_js//...`?

Reply:
That test fails though.

Reply:
Maybe you can be more specific on which targets should build (instead of `@aspect_rules_js//..`)?